dev-manuals/backend/dapr/dapr-topics.md

```diff
@@ -18,7 +18,8 @@ This includes
 | Content Service    | [chapter-changes](#topic-chapter-changes)<br>[content-progressed](#topic-content-progressed)<br> [resource-update](#topic-resource-update) | [resource-association](#topic-resource-association)<br>[content-changes](#topic-content-changes) |
 | Media Service      | [content-changes](#topic-content-changes)                                                                                                  | [resource-update](#topic-resource-update)<br>[content-progressed](#topic-content-progressed)     |
 | Flashcard Service  | [content-changes](#topic-content-changes)                                                                                                  | [resource-update](#topic-resource-update)    <br>[content-progressed](#topic-content-progressed) |
-| Reward Service     | [content-progressed](#topic-content-progressed)                                                                                            |                                                                                                  |
+| Quiz Service       | [content-changes](#topic-content-changes)                                                                                                  | [content-progressed](#topic-content-progressed)                                                  |
+| Reward Service     | [content-progressed](#topic-content-progressed)<br>[course-changes](#topic-course-changes)                                                 |                                                                                                  |
```

Comment:
this isnt even up to date but that is my fault not yours

Reply:
Oh yeah?! I'll check up on all services then

Reply:
oh thanks but I could also do that